book/security.rst

```diff
@@ -1043,6 +1043,17 @@ if your User object has a ``getFirstName()`` method, you could use that::
         return new Response('Well hi there '.$user->getFirstName());
     }
 
+.. versionadded:: 2.8
+     The ``security.current_user_provider`` service was introduced in Symfony 2.8. It makes it 
+     easier to retrieve the current logged in user from a service. 
+     
+    .. code-block:: php
+
+        $user = $this->container->get('security.current_user_provider')->getUser();
+        if ($user === null) {
```

Comment:
(1) Yes, it should be switched indeed.
(2) I agree with @javiereguiluz here, I've left a comment in the code PR.